NitroxModel/Discovery/GameInstallationFinder.cs

```diff
@@ -13,8 +13,6 @@ public class GameInstallationFinder : IFindGameInstallation
     {
         private readonly List<IFindGameInstallation> finders = new List<IFindGameInstallation>
         {
-            new SteamGameRegistryFinder(),
-            new EpicGamesRegistryFinder(),
```

Comment:
Why were these removed?

Reply:
uhh sorry that shouldn't be there... i have to do that as it causes problems since i have a few different installs. Removing.